lib/utils.js

```diff
@@ -76,15 +76,6 @@ const findLocationOf = (keys, ast, initialFormat) => {
   };
 };
 
-const traverse = function (o, fn, scope = []) {
```

Comment:
no need to traverse anymore, json ref parser now gives the information about where the error happened